save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/SelectForm.kt

```diff
@@ -109,6 +109,7 @@ fun <D : Any> selectFormRequired() = FC<SelectFormRequiredProps<D>> { props ->
                 className = ClassName("form-control")
                 id = props.formType.name
                 required = true
+                // TODO: why we need an extra option
```

Comment:
I believe it's an empty option which is selected by default

Reply:
probably, but there is `value = props.selectedValue`. maybe we need to add an empty value only if `props.selectedValue` is not presented in `elements`. Need to discuss with @sanyavertolet